tests/integration-tests/conftest.py

```diff
@@ -908,3 +909,31 @@ def _pcluster_ami_without_standard_naming(version):
     if ami_id:
         client = boto3.client("ec2", region_name=region)
         client.deregister_image(ImageId=ami_id)
+
+
+@pytest.fixture()
+def build_image():
+    """Create a build image process."""
+    image_name_post_test = None
+    region_post_test = None
+
+    def _build_image(image_name, region, cluster_config):
+        nonlocal image_name_post_test
+        nonlocal region_post_test
+        image_name_post_test = image_name
+        region_post_test = region
+
+        pcluster_build_image_result = run_command(
+            ["pcluster", "build-image", "-n", image_name, "-r", region, "-c", cluster_config.as_posix()]
```

Comment:
this needs to be updated accordingly to https://github.com/aws/aws-parallelcluster/pull/2711

Reply:
done